docs/modules/ROOT/pages/reference.adoc

```diff
@@ -146,3 +146,20 @@ Appears In: xref:{anchor_prefix}-github-com-hacbs-contract-enterprise-contract-c
 
 
 
+[id="{anchor_prefix}-github-com-hacbs-contract-enterprise-contract-controller-api-v1alpha1-source"]
+==== Source
+
+Source defines policies and data that are evaluated together
+
+[quote]
+Appears In: xref:{anchor_prefix}-github-com-hacbs-contract-enterprise-contract-controller-api-v1alpha1-enterprisecontractpolicyspec[$$EnterpriseContractPolicySpec$$]
+
+[cols="25a,75a", options="header"]
+|===
+| Field | Description
+| *`name`* __string__ | Name of the source
```

Comment:
Is the name optional? If yes should we mention that, e.g. "Name of the source (optional)" or "Optional name for this source"?

Reply:
Went with the second suggestion, and also marked the field with `+optional`.